apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -50,12 +60,25 @@ domReady( () => {
 			newCloseButton.prepend( thinContent );
 		} else if ( 'page' === editorPostType ) {
 			newCloseButton.href = 'edit.php?post_type=page';
-			const newLabel = __( 'Back to Page List' );
+			const newLabel = __( 'Pages' );
+			newCloseButton.setAttribute( 'aria-label', newLabel );
+
+			ReactDOM.render(
+				<Button className="components-button components-icon-button">
```

Comment:
What if we just did one render at the bottom and conditionally assign variables based on the post type?

Reply:
Yeah that works.  I initially did not do this because the template_part does not use that render, but added a condition for page/post so we only have to call .render() in one area.